src/signals/incident/components/IncidentForm/index.test.js

```diff
@@ -285,6 +285,97 @@ describe('<IncidentForm />', () => {
 
         expect(nextSpy).toHaveBeenCalledTimes(2)
       })
+
+      it('should postpone submit when postponeSubmitWhenLoading is defined for loadingQuestions', () => {
+        const props = {
+          ...defaultProps,
+          postponeSubmitWhenLoading: 'loadingQuestions',
+        }
+
+        const { rerender } = renderIncidentForm({
+          ...props,
+          loadingQuestions: true,
+        })
+
+        expect(nextSpy).toHaveBeenCalledTimes(1)
+        userEvent.click(screen.getByText(mockForm.nextButtonLabel))
+        expect(nextSpy).toHaveBeenCalledTimes(1)
+
+        renderIncidentForm(props, rerender)
+
+        expect(nextSpy).toHaveBeenCalledTimes(2)
+      })
+
+      it('should postpone submit when postponeSubmitWhenLoading is defined for both loadingClassification and loadingQuestions', () => {
```

Comment:
Same here: Does this mean: "it postpones submit until the classification and questions have loaded" ?